api/Dockerfile

```diff
@@ -43,7 +46,11 @@ RUN cd /tmp && \
 
 WORKDIR /app
 COPY ./app /app/app
+RUN mkdir /app/libs
+# COPY ./libs/hss-java.jar ./libs/REDapp_Lib.jar ./libs/WTime.jar /app/libs/
```

Comment:
did you intend to un-comment this line or delete it?

Reply:
Thanks - I'll kill it.